app/workspace/page.tsx

```
@@ -282,168 +597,344 @@ function WorkspaceContent() {
+                    All Documents ({documents.length})
+                  </h2>
+                  
+                  <GlassCard className="overflow-hidden">
+                    <div className="divide-y divide-leather-700/30">
+                      {documents.map((doc) => (
+                        <div
+                          key={doc.id}
+                          onClick={() => handleOpenDocument(doc.id)}
+                          className="p-4 hover:bg-leather-900/30 cursor-pointer transition-colors group"
+                        >
+                          <div className="flex items-center gap-4">
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)